packages/react/src/button/Button.stories.tsx

```diff
@@ -19,7 +19,6 @@ const meta: Meta<typeof Button> = {
 		block: false,
 		children: 'Button',
 		disabled: false,
-		loading: false,
```

Comment:
Although this gives 1-click access to a loading control, it is also not applicable for the ButtonLink story, so it was easier to just remove it. Now it's a 2-click access to a loading control

Reply:
Does that need to be indicated somewhere?

Reply:
We should probably refactor the stories slight to separate Button and ButtonLink ones… after the audit